dlc-manager/src/manager.rs

```diff
@@ -129,7 +129,10 @@ macro_rules! check_for_timed_out_channels {
                 let is_timed_out = timeout < $manager.time.unix_time_now();
                 if is_timed_out {
                     let sub_channel = if channel.is_sub_channel() {
-                        unimplemented!();
+                        error!("Force-closing subchannels is not supported; skipping closure.");
```

Comment:
To be more precise, what's not supported is force-closing _timed out_ subchannels, but AFAIK we could force-close here if we wanted to.

In 10101 we probably want more control over triggering a force-close, so I think we just don't want to call `check_for_timed_out_channels` at all. I wonder if this feature of the library is actually useful for any consumer, since force-closing is often an expensive decision cc @Tibo-lg. AFAIK this timeout is not based on the refund timelock, so we're being overeager.

I agree with removing the unimplemented, but I would maybe just log on Debug or nothing at all.